### PR TITLE
[Security] translations for Tagalog (tl)

### LINKED ISSUE #42473: Missing translations for Tagalog (tl)
Hello,

There are some missing translation for Tagalog (tl). These should be added in branch 4.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Security/Core/Resources/translations/security.tl.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Security/Core/Resources/translations/security.tl.xlf)
- [src/Symfony/Component/Validator/Resources/translations/validators.tl.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Validator/Resources/translations/validators.tl.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by shreyadenny**
| Q             | A
| ------------- | ---
| Branch?       | 4.4 
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Tickets       | Fix #42473 
| License       | MIT

Added translations for Tagalog(tl)

### CODE REVIEW

**Comment by fabpot** [comment]
Thank you for working on this. I can see that you've improved the current translations, but can you also add the missing ones?

**Comment by shreyadenny** [comment]
> Thank you for working on this. I can see that you've improved the current translations, but can you also add the missing ones?

Hey @fabpot

I have added all translations after comparing with <a href="https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Security/Core/Resources/translations/security.en.xlf">this</a>

Could you tell me which are the missing translations that you are referring to?

**Comment by fabpot** [comment]
There is another file here: https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Validator/Resources/translations/validators.en.xlf

**Comment by shreyadenny** [comment]
> There is another file here: https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Validator/Resources/translations/validators.en.xlf

Sure thanks. I'm on it.

**Comment by shreyadenny** [comment]
@fabpot I've added the missing translations. Please check and suggest changes if needed.

**Comment by fabpot** [comment]
Thank you for the work, but I think there was a misunderstanding. There are 2 different files for each language. Here, you've mixed up the 2 files into 1.

**Comment by shreyadenny** [comment]
Sorry I may have made a mistake. I am correcting it now . Thanks for your help

**Comment by shreyadenny** [comment]
I have made some corrections. Hopes it's okay now. @fabpot

File: src/Symfony/Component/Security/Core/Resources/translations/security.tl.xlf [hunk 1]
```diff
@@ -28,56 +28,56 @@
...
-                <target>Madaming bagsak na pagtatangka, pakisubukan ulit pagkatapos ng %minutes% minuto.</target>
+                <target>Napakaraming nabigong mga pagtatangka sa pag-login, pakisubukan ulit sa% minuto% minuto.</target>
             </trans-unit>
             <trans-unit id="20">
                 <source>Too many failed login attempts, please try again in %minutes% minute.</source>
-                <target>Madaming bagsak na pagtatangka, pakisubukan ulit pagkatapos ng %minutes% minuto.</target>
+                <target>Napakaraming nabigong mga pagtatangka sa pag-login, pakisubukan ulit sa% minuto% minuto.</target>
             </trans-unit>
-        </body>
+                    </body>
```
Comment (fabpot) [inline src/Symfony/Component/Security/Core/Resources/translations/security.tl.xlf]: Can you revert this change?

File: src/Symfony/Component/Validator/Resources/translations/validators.th.xlf [hunk 2]
```diff
@@ -390,6 +390,7 @@
                 <source>This value should be a valid expression.</source>
                 <target>ค่านี้ควรเป็นนิพจน์ที่ถูกต้อง</target>
             </trans-unit>
+
```
Comment (fabpot) [inline src/Symfony/Component/Validator/Resources/translations/validators.th.xlf]: Can you revert this change?

**Review by fabpot — CHANGES_REQUESTED** [review]

**Comment by shreyadenny** [comment]
@fabpot Yes I've don the changes

**Pull request closed without merging**